src/System.Management.Automation/resources/LocationStrings.resx

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think it's reasonable to put the new string in `SessionStateStrings` instead of this new file.

Reply:
Ok.